pkgs/by-name/ot/otree/package.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+, stdenv
+, git
+, darwin
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "otree";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "fioncat";
+    repo = "otree";
+    rev = "v${version}";
+    hash = "sha256-ukKlypGY0+xsci/loAnf0Sz5K+82gVzkb+KnZ139cIA=";
+    # FIXME: this should no longer be needed in the next release.
+    leaveDotGit = true; # needed by build.rs
+  };
+
+  # TODO: remove the git dependency via use of BUILD_OTREE_WITH_GIT_INFO and CARGO_PKG_VERSION for the next release.
+  nativeBuildInputs = [ git ];
+
+  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdks.frameworks.IOKit;
```

@0x61nas My previous suggestion contained a typo (it's `apple_sdk` not `apple_sdks`), apologies for that…

```suggestion
  buildInputs = lib.optional stdenv.isDarwin darwin.apple_sdk.frameworks.IOKit;
```

---

oh, ok

---

NP, fixed